Externals.cfg

```diff
@@ -50,7 +50,7 @@ protocol = git
 required = False
 
 [GDASApp]
-hash = 843d3a9
+hash = e27973d
```

Comment:
Note:  We may need to update the GDASApp hash before this PR is merged into g-w develop.